jwst/extract_2d/grisms.py

```diff
@@ -383,7 +383,7 @@ def extract_grism_objects(input_model,
                 # set x/ystart values relative to the image (screen) frame.
                 # The overall subarray offset is recorded in model.meta.subarray.
                 # nslit = obj.sid - 1  # catalog id starts at zero
-                new_slit.name = "Object {0}".format(obj.sid)
+                new_slit.name = "Object_{0}".format(obj.sid)
```

Comment:
I wonder if we should make the name syntax here consistent with what we're doing for NRS MSA data, i.e. something that's just an integer without "Object_"?

Reply:
I agree. 👍 for less typing